extensions/amp-analytics/0.1/amp-analytics.js

```diff
@@ -220,6 +225,21 @@ export class AmpAnalytics extends AMP.BaseElement {
   }
 
   /**
+   * @param {!JSONType} config
+   * @private
+   */
+  addTriggerNoInline_(config) {
```

Comment:
Sure. Will add description. FYI, NoInline suffix ensures that closure doesn't inline the method. Since we have `try/catch` inlining it will cause the calling methods to loose some optimizations.